core/iwasm/aot/aot_runtime.c

```diff
@@ -2501,6 +2501,12 @@ aot_table_init(AOTModuleInstance *module_inst, uint32 tbl_idx,
     AOTTableInstance *tbl_inst;
     AOTTableInitData *tbl_seg;
     const AOTModule *module = (AOTModule *)module_inst->module;
+#if WASM_ENABLE_GC != 0
+    table_elem_type_t **table_elems;
```

Comment:
Shold be `table_elem_type_t *table_elems`, table_elem_type_t is `void *` or `uintptr_t` when GC is enabled or not.